uk.co.compucorp.civicrm.hremails/email-templates/src/assets/scss/components/_request.scss

```diff
@@ -15,12 +15,17 @@
 
 
 .request-comment {
-  padding: $callout-padding;
+
```

Comment:
I can see this empty line is commonly used in projects, is it a standard or style guide that we should follow? I didn't find this mentioned somewhere and even the examples i found for nested elements doesn't include this empty line. 

https://css-tricks.com/sass-style-guide/ (check 'Maximum Nesting: Three Levels Deep' example)